module/go.mod

```diff
@@ -3,26 +3,26 @@ module github.com/Gravity-Bridge/Gravity-Bridge/module
 go 1.16
 
 require (
```

Comment:
Notable updates: cosmos-sdk, ibc-go
After ibc-go updated we needed to update bech32ibc and tharsis/ethermint (ethermint 0.9 depended on ibc v2 and caused issues)